tests/scripts/task_python_microtvm.sh

```diff
@@ -45,9 +45,10 @@ run_pytest ctypes python-microtvm-stm32 tests/micro/stm32
 run_pytest ctypes python-microtvm-common-qemu_x86 tests/micro/common --board=qemu_x86
 run_pytest ctypes python-microtvm-common-due tests/micro/common  --test-build-only --board=due
 
-# # Tutorials running with host CRT
+# Tutorials
 python3 gallery/how_to/work_with_microtvm/micro_tflite.py
 python3 gallery/how_to/work_with_microtvm/micro_autotune.py
+./gallery/how_to/work_with_microtvm/micro_tvmc.sh
```

Comment:
can we add this to `make docs` instead?

Reply:
This script requires zephyr.

Reply:
I think that would be a bigger change. GPU docker image which is used for doc generation should be able to run Zephyr and other platforms in future. Other possibility is to have each docker step to run and convert the script which is related to their target and store the output in a shared directory which is passed to GPU doc stage.

Reply:
this is almost what we do now (the GPU docker image runs the tutorials and sends a `tar.gz` to another step to deploy it), we could do the same here and add some way of unifying the two (like unpack, make sure no file names collide, send the whole thing to the docs deploy)

Reply:
@driazati  Does this work between executors? Can we run something on ci-qemu and pass it to ci-gpu for doc generation?

Reply:
k, so for this PR the delta is at least that now we test the tvmc commands to make sure they don't crash. next it would be great to do what @driazati mentioned, where we run script_convert.py in ci-qemu and then forward the output to the doc-builder step. let's merge this now though to make incremental progress.